specs/data_structures.md

```diff
@@ -161,11 +171,9 @@ https://developers.google.com/protocol-buffers/docs/proto3
 
```

Comment:
I think the header of this section should be called `# Wire format`. This is only about data serialization over the network. E.g. formats of data signed, or stored can be different (e.g. JSON, sth canonicalized). While the spec needs to define how signed data gets signed, the implementer is free to chose whatever they like for storage.

Reply:
Hmm I was thinking this section was more defining a canonical serialization format, for use with signing and hashing, etc. Data is transferred over the wire could be different, and should be in a separate networking document.